.gitignore

```diff
@@ -0,0 +1,2 @@
+node_module
+README.md
```

Comment:
you perfectly added .gitignore but it also needs one blank line at the end of the file.

Reply:
> you perfectly added .gitignore but it also needs one blank line at the end of the file.

- [ ] It is always important to add the WHY, this is for the student to know the reason you are requesting changes.